cli/compose/schema/data/config_schema_v3.14.json

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
Schema 3.13 was added in https://github.com/docker/cli/pull/5125, which has not been released yet, and will be for the v27.0 release, so we don't  need to add a new schema and can update 3.13 instead.